packages/app/src/cli/services/app/config/link.ts

```diff
@@ -69,12 +69,23 @@ async function loadRemoteApp(localApp: AppInterface, apiKey: string | undefined)
 }
 
 function mergeAppConfiguration(localApp: AppInterface, remoteApp: OrganizationApp): AppConfiguration {
-  const configuration = localApp.configuration || {}
+  if (localApp?.configuration) {
+    const configuration = localApp.configuration as CurrentAppConfiguration
 
-  configuration.client_id = remoteApp.apiKey
-  configuration.name = remoteApp.title
-  configuration.application_url = remoteApp.applicationUrl
-  configuration.redirect_url_allowlist = remoteApp.redirectUrlWhitelist
+    configuration.client_id = remoteApp.apiKey
```

Comment:
this needs to be updated to include the full schema, but i wanted to move `use` and `link` updates to a new pr